src/components/LocationAutocomplete/LocationAutocompleteRenderer/LocationAutocompleteRenderer.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import { Autosuggest } from '../../Autosuggest';
+import useDebounce from '../../../hooks/useDebounce';
+import { ListItem, MarkedText } from '../../../index';
+import POWERED_BY_GOOGLE_ON_WHITE from '../../../images/powered_by_google_on_white.png';
+import styles from '../LocationAutocomplete.scss';
+
+const { elem } = bem({
+    name: 'LocationAutocomplete',
+    classnames: styles,
+});
+
+const DEBOUNCE_DELAY = 350;
+
+const LocationAutocomplete = props => {
+    const [inputValue, setInputValue] = React.useState('');
+    const [suggestionsList, setSuggestionsList] = React.useState([]);
+    const [isLoading, setIsLoading] = React.useState(false);
+
+    const {
+        onSelectionChange,
+        inputPlaceholder,
+        noSuggestionsPlaceholder,
+        country,
+        placeTypes,
+        ...rest
+    } = props;
+
+    // Suggestion functions
+    const resetSuggestionsList = () => setSuggestionsList([]);
+    const suggestionToString = suggestion => (suggestion ? suggestion.description : '');
+
+    const debouncedInputValue = useDebounce(inputValue, DEBOUNCE_DELAY);
+
+    React.useEffect(() => {
+        if (debouncedInputValue) {
+            const service = new window.google.maps.places.AutocompleteService();
+
+            service.getPlacePredictions(
+                {
+                    input: debouncedInputValue,
+                    types: placeTypes,
+                    componentRestrictions: { country },
+                },
+                predictions => {
+                    setSuggestionsList(predictions);
+                    setIsLoading(false);
+                }
+            );
```

Comment:
We can't 100% rely on external API, so better to wrap it in **try {} catch(err){}**

Reply:
added some provisional error handling

Reply:
if Google doesn't expose on some reasons window.google.... object, or it will be inited with a delay so in this case, UI can go down and we got the white screen.